Source/Krypton Components/Krypton.Toolkit/ButtonSpec/ButtonSpecFormWindowClose.cs

```diff
@@ -60,18 +60,8 @@ public bool Enabled
         /// </summary>
         /// <param name="palette">Palette to use for inheriting values.</param>
         /// <returns>Button visibility.</returns>
-        public override bool GetVisible(PaletteBase? palette)
-        {
-            // We do not show if the custom chrome is combined with composition,
-            // in which case the form buttons are handled by the composition
-            if (KryptonForm is { ApplyComposition: true, ApplyCustomChrome: true })
-            {
-                return false;
-            }
-
-            // Have all buttons been turned off?
-            return KryptonForm is { ControlBox: true, CloseBox: true };
-        }
+        public override bool GetVisible(PaletteBase? palette) =>
```

Comment:
I did not see these changes in the V95 version.
Also update the file Year change please